epm/pkg/epm/enclave-cache-pool/enclavepool/enclave-cache.go

```diff
@@ -62,18 +62,25 @@ func (d *EnclaveCacheManager) GetPoolType() string {
 	return d.Type
 }
 
+func SaveFd(cacheID string, err *error) {
+	var fd int
+
+	sockpath := "/var/run/sock/" + cacheID
```

Comment:
It's better to import `"path/filepath"` and use `filepath.Join` to get the absolute path.

Reply:
ok, I will change, thanks